vocabularies/geoscience-units-of-measurement.ttl

```diff
@@ -3899,17 +4219,27 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
         "unkn"@en ,
         "unknown"@en ;
     skos:definition "The unit for a given value is unknown."@en ;
-    skos:inScheme cs: ;
+    skos:inScheme <https://linked.data.gov.au/def/uom-geoscience> ;
     skos:prefLabel "unknown"@en ;
-    skos:topConceptOf cs: ;
+    skos:topConceptOf <https://linked.data.gov.au/def/uom-geoscience> ;
+    schema:status <https://linked.data.gov.au/def/reg-statuses/stable> ;
 .
 
-:1.2 rdfs:label "1.2" .
+<https://orcid.org/0009-0002-5543-2719>
+    a schema:Person ;
+    schema:email ""^^xsd:anyURI ;
+    schema:name "0009 0002 5543 2719" ;
+.
+
+<https://linked.data.g​ov.au/org/gswa>
+    a schema:Organization ;
+    schema:name "Gswa" ;
+    schema:url ""^^xsd:anyURI ;
+.
```

Comment:
```suggestion
```

Remove these to fix the validation error as they are already defined in [background/agents.ttl](https://github.com/Geological-Survey-of-Western-Australia/Vocabularies/blob/main/background/agents.ttl).